toolchain/check/import.cpp

```diff
@@ -89,53 +89,69 @@ static auto CopyNameFromImportIR(Context& context,
   return import_name_id;
 }
 
+static auto MakeImportedNamespaceLocIdAndInstId(
```

Comment:
This overlaps with what I'm doing in #5120 (where I'm trying to remove the `UnsafeMake`), so there may be a collision. That said, the `Id` suffix doesn't match the return type, so maybe:

```suggestion
// Returns the LocIdandInst for the namespace.
static auto MakeImportedNamespaceLocIdAndInst(
```

Reply:
Thanks for the heads up!
Done.